filters.go

```diff
@@ -134,3 +134,26 @@ func (p *Pipe) Concat() *Pipe {
 	})
 	return p.WithReader(io.MultiReader(readers...))
 }
+
+// First reads from the pipe, and returns a new pipe containing only the first N
+// lines. If there is an error reading the pipe, the pipe's error status is also
+// set.
+func (p *Pipe) First(lines int) *Pipe {
```

Comment:
Currently negative values are treated the same as 0. `head` has a special interpretation of negative numbers, which could be added here as well.

Reply:
> GNU head allows negative arguments for -n option, meaning to print all but the last - argument value counted - lines of each input file

That sounds like effectively it would behave like `Last()`—well, we don't have that yet, but when we do, we won't need to worry about this :)

Reply:
No, it is not like `Last()`, because it prints everything BUT the last n lines, where `Last()` prints ONLY the last n lines. So in fact it is more like an inverted `Last()`.
A similar option is available in `tail` als well.

Reply:
Oh yes, I wasn't reading that properly. Well, it's a neat feature, but I don't have a use case for it—so I'll wait until there is one.